src/context/ThemeContext.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const ThemeContext = createContext('rgba(10, 10, 10, 0.5)');
```

Comment:
We don't need to have color rgba as context value. It would be more descriptive if values would be 'dark' & 'light'. In that way if you change the design (colors of themes), you change them in fewer places in code and you don't have to compare rbga values to identify current theme.